homeassistant/components/zone.py

```diff
@@ -90,26 +92,45 @@ def setup(hass, config):
                     'Each zone needs a latitude and longitude.')
                 continue
 
-            zone = Zone(hass, name, latitude, longitude, radius, icon, passive)
+            zone = Zone(hass, name, latitude, longitude, radius, icon,
+                        passive, False)
             zone.entity_id = generate_entity_id(ENTITY_ID_FORMAT, name,
                                                 entities)
             zone.update_ha_state()
             entities.add(zone.entity_id)
 
     if ENTITY_ID_HOME not in entities:
         zone = Zone(hass, hass.config.location_name, hass.config.latitude,
-                    hass.config.longitude, DEFAULT_RADIUS, ICON_HOME, False)
+                    hass.config.longitude, DEFAULT_RADIUS, ICON_HOME,
```

Comment:
I am getting a `too-many-arguments` lint error if I just add all of Zone parameters to the add_zone call. So I will create a Zone outside of add_zone and use add_zone only for `genrate_entity_id` and `update_ha_state`